includes/Core/Authentication/Authentication.php

```diff
@@ -1467,9 +1467,11 @@ public function invalid_nonce_error( $action ) {
 		$html  = __( 'The link you followed has expired.', 'google-site-kit' );
 		$html .= '</p><p>';
 		$html .= sprintf(
-			'<a href="%s">%s</a>. <a href="%s" target="_blank">%s</a>.',
+			/* translators: 1: Admin splash URL. 2: Try again string. 3: Retry string. 4: Support link URL. 5: Get help string. */
+			__( '<a href="%1$s">%2$s</a>. %3$s <a href="%4$s" target="_blank">%5$s</a>.', 'google-site-kit' ),
```

Comment:
Looking at similar strings in core, placeholders should probably only be used for URLs rather than all of the bits text which more/less goes against the rules for concatenating translated strings. Core isn't 100% consistent about this either but I think it makes sense to follow in this case.

<img width="871" alt="image" src="https://user-images.githubusercontent.com/1621608/194937477-56b4e203-1f2d-48e1-ae3e-4b30c7030c64.png">

Let's move everything except the URLs into this larger string instead.

Reply:
Updated, thank you @aaemnnosttv!